backend/dataall/modules/dashboards/api/resolvers.py

```diff
@@ -0,0 +1,143 @@
+from dataall.api.context import Context
+from dataall.db import models
+from dataall.db.api import  Glossary, Vote
+from dataall.db.exceptions import RequiredParameter
+from dataall.modules.dashboards.api.enums import DashboardRole
+from dataall.modules.dashboards.db.dashboard_repository import DashboardRepository
+from dataall.modules.dashboards.db.models import Dashboard
+from dataall.modules.dashboards.services.dashboard_quicksight_service import DashboardQuicksightService
+from dataall.modules.dashboards.services.dashboard_service import DashboardService
+from dataall.modules.dashboards.services.dashboard_share_service import DashboardShareService
+
+
+def import_dashboard(context: Context, source, input: dict = None):
+    if not input:
+        raise RequiredParameter(input)
+    if not input.get('environmentUri'):
+        raise RequiredParameter('environmentUri')
+    if not input.get('SamlGroupName'):
+        raise RequiredParameter('group')
+    if not input.get('dashboardId'):
+        raise RequiredParameter('dashboardId')
+    if not input.get('label'):
+        raise RequiredParameter('label')
+
+    return DashboardService.import_dashboard(uri=input['environmentUri'], data=input)
+
+
+def update_dashboard(context, source, input: dict = None):
+    return DashboardService.update_dashboard(uri=input['dashboardUri'], data=input)
```

Comment:
you could pass here just the `input` as 1 function argument

Reply:
Unfortunately, we couldn't. Uri is used for the resources permission checker.